src/file-selector.ts

```diff
@@ -194,6 +215,16 @@ const readEntries = (
 ): Promise<FileSystemEntry[]> =>
   new Promise((resolve, reject) => reader.readEntries(resolve, reject));
 
+declare global {
```

Comment:
Would this cause issues in downstream packages if they have different declarations? I'm not up to date with how the `declare global` works and if it crosses the boundary b/t packages (if 2 packages declare the same in the global namespace).